components/suggestion/search_channel_suggestion/search_channel_suggestion.tsx

```diff
@@ -3,22 +3,23 @@
 import React from 'react';
 
 import {getUserIdFromChannelName} from 'mattermost-redux/utils/channel_utils';
-
 import {imageURLForUser} from 'utils/utils.jsx';
 import Constants from 'utils/constants';
 import Avatar from 'components/widgets/users/avatar';
 import BotBadge from 'components/widgets/badges/bot_badge';
+import Suggestion from '../suggestion';
 
-import Suggestion from '../suggestion.jsx';
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Channel} from 'mattermost-redux/types/channels';
 
-function itemToName(item, currentUser) {
-    let itemMarkup = (item);
+function itemToName(item: Channel, currentUser: UserProfile) {
```

Comment:
Hi @devinbinnie , I set the data type for param currentUser (previously string in jsx file) to be  "UserProfile".